Objects/call.c

```diff
@@ -1276,45 +1276,19 @@ PyObject_CallFunctionObjArgs(PyObject *callable, ...)
 _Py_NO_INLINE PyObject *
 _PyStack_AsTuple(PyObject *const *stack, Py_ssize_t nargs)
```

Comment:
@vstinner 
Does it make sense to have `_PyStack_AsTuple()` and `_PyStack_AsTupleSlice()` after adding `PyTuple_FromArray()`? Might make sense to remove them or replace with macros.

Reply:
You can remove _PyStack_AsTuple, but please do that in a second PR (once this one is merged, if it's merged ;-)).

Reply:
I would remove both `_PyStack_AsTuple` and `_PyStack_AsTupleSlice`. The former is just an alias of `_PyTuple_FromArray`, and the latter is an alias with trivial arguments transformation, and is used only once.